src/draw/sw/blend/lv_draw_sw_blend.h

```diff
@@ -53,6 +53,7 @@ typedef struct {
     int32_t mask_stride;
     lv_color_t color;
     lv_opa_t opa;
+    lv_area_t realitive_area;
```

Comment:
@kisvegabor should be `relative_area` instead?

Reply:
Yes :sweat_smile: